modules/ppcp-settings/src/SettingsModule.php

```diff
@@ -115,6 +118,7 @@ public function run( ContainerInterface $container ): bool {
 					esc_html__( 'This action will permanently switch to the new settings interface and cannot be undone', 'woocommerce-paypal-payments' )
 				)
 			);
+			*/
```

Comment:
But why disabling for everyone? Shouldn't we disable it only for ACDC merchants who has BCDC enabled?